packages/flutter_markdown/lib/src/widget.dart

```diff
@@ -70,13 +70,33 @@ abstract class MarkdownElementBuilder {
   /// If you needn't build a widget, return null.
   Widget? visitText(md.Text text, TextStyle? preferredStyle) => null;
 
+  /// Called when an Element has been reached, after its children have been
+  /// visited.
+  ///
+  /// If [MarkdownWidget.styleSheet] has a style of this tag, will passing
```

Comment:
This doesn't parse as a sentence. I'm assuming this should say something like "has a style for this tag, it will be passed as [preferredStyle]."?